src/test/java/core/basesyntax/service/strategy/PurchaseOperationHandlerTest.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.service.strategy;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import org.junit.AfterClass;
+import org.junit.Test;
+
+public class PurchaseOperationHandlerTest {
+    @Test
+    public void getCorrectPurchaseOperation_Ok() {
+        Storage.fruits.put("banana", 100);
+
+        FruitTransaction fruitTransaction = new FruitTransaction();
+        fruitTransaction.setFruit("banana");
+        fruitTransaction.setQuantity(50);
+
+        OperationHandler operationHandler = new PurchaseOperationHandler();
```

Comment:
Same. The services to be tested should be written at class level. Fix everywhere